pkg/disk/rpcserver/nodeserver.go

```diff
@@ -525,3 +556,37 @@ func (ns *NodeServer) getBlockSizeBytes(devicePath string) (int64, error) {
 	}
 	return gotSizeBytes, nil
 }
+
+// createTargetMountPathIfNotExists creates the mountPath if it doesn't exist
+// if in block volume mode, a file will be created
+// otherwise a directory will be created
+func (ns *NodeServer) createTargetMountPathIfNotExists(mountPath string, isBlockMode bool) error {
+	exists, err := ns.mounter.ExistsPath(mountPath)
+	if err != nil {
+		klog.Errorf("Failed to determine whether mountPath:%s exists: %v", mountPath, err)
+		return err
+	}
+	if exists {
+		return nil
+	}
+
+	if isBlockMode {
+		//#nosec
+		pathFile, err := os.OpenFile(mountPath, os.O_CREATE|os.O_RDWR, 0750)
+		if err != nil {
+			klog.Errorf("Failed to create mountPath:%s with error: %v", mountPath, err)
+			return err
+		}
+		if err = pathFile.Close(); err != nil {
+			klog.Errorf("Failed to close mountPath:%s with error: %v", mountPath, err)
```

Comment:
almost all errors in this function were logged. This is not recommended. Should return the error, and let the caller decide whether to log the error.